charts/graphistry-helm/templates/dask/dask-scheduler-deployment.yaml

```diff
@@ -0,0 +1,136 @@
+{{ if eq .Values.dask.operator false }}
+{{ if eq .Values.global.devMode false }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    io.kompose.service: dask-scheduler
+  name: dask-scheduler
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      io.kompose.service: dask-scheduler
+  strategy:
+{{ if eq .Values.rollingUpdate true }}
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: {{.Values.maxSurge}}
+      maxUnavailable: "0%"
+{{ else }}
+    type: Recreate
+{{- end }}
+  template:
+    metadata:
+      labels:
+        io.kompose.network/grph: "true"
+        io.kompose.service: dask-scheduler
+    spec:
+      initContainers:
+      - name: "{{ lower .Chart.Name }}-init"
+        {{- if eq .Values.global.containerregistry.name "docker.io/graphistry" }}
+        image: "groundnuty/k8s-wait-for:latest" #DockerHub
+        {{ else }}
+        image: {{.Values.global.containerregistry.name}}/k8s-wait-for:latest
+        {{- end }}
+        resources:
+            {{- toYaml .Values.InitContainerResources | nindent 10 }}
+        imagePullPolicy: {{.Values.global.imagePullPolicy }} 
+        args: 
+        - "service"
+        - "nexus"
+      containers:
+        - args:
+            - dask-scheduler
+            - --port
+            - "8786"
+            - --interface
+            - eth0
+            - --no-show
+            - --dashboard-address
+            - "8787"
+          command:
+            - /entrypoints/rapids-entrypoint.sh
+          env:
```

Comment:
@webcoderz There are some handy env variables for configuring dask scheduler too, you can take a look here in case we need to set some of these:
https://docs.dask.org/en/latest/configuration.html#environment-variables